scripts/custom_sql_query/run_custom_query.py

```diff
@@ -226,6 +236,18 @@ def get_current_user():
         sys.exit(1)
 
 
+def get_db_endpoint(environment):
+    instance_id = f"api-{environment}-0"
+    rds = boto3.client("rds")
+    try:
+        response = rds.describe_db_instances(DBInstanceIdentifier=instance_id)
+        db_instance = response["DBInstances"][0]
```

Comment:
Is it always going to be the first instance in the list? Is there any way to be more specific in finding the db?

Reply:
It's based on the instanceid which is always 0 for the writer instance. It's set here :) 
`instance_id = f"api-{workspace}-0"`